### PR TITLE
Property '_lg' now holds the user's language

### LINKED ISSUE #10: The `_lg` property should contain the Application language and not the device language
To get the application language:
`properties.put(APP_LOCALE, mContext.getResources().getConfiguration().locale.toString());`

Put the device language as a new device property. Current devices properties below:
```
            [device_info_os] => Android
            [device_info_os_version] => 5.0
            [device_info_manufacturer] => Genymotion
            [device_info_brand] => generic
            [device_info_model] => Google Nexus 6 - 5.0.0 - API 21 - 1440x2560
            [device_info_app_name] => WordPress
            [device_info_app_version] => 4.1-rc-1
            [device_info_app_version_code] => 182
            [device_info_has_nfc] => false
            [device_info_has_telephony] => true
            [device_info_display_density_dpi] => 560
            [device_info_bluetooth_version] => none
            [device_info_current_network_operator] => 
            [device_info_phone_radio_type] => gsm
            [device_info_wifi_connected] => true
```

/cc @astralbodies

### PR DESCRIPTION
**Pull request by daniloercoli**
Fix #10

### CODE REVIEW

File: AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java [hunk 1]
```diff
@@ -125,6 +125,7 @@ public DeviceInformation(Context context) {
             mImmutableDeviceInfoJSON.put("app_name", getAppName());
             mImmutableDeviceInfoJSON.put("app_version", getAppVersionName());
             mImmutableDeviceInfoJSON.put("app_version_code", Integer.toString(getAppVersionCode()));
+            mImmutableDeviceInfoJSON.put("language", getDeviceLanguage());
```
Comment (maxme) [inline AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java]: some fields here are prefixed with `app_`, should we name that field `device_language` so it's obvious that's not the app language?

Comment (daniloercoli) [inline AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java]: All the fields returned from `DeviceInformation` are prefixed with `device_info` when sent to the server. Resulting in something like the following structure:
```
           [device_info_os] => Android
           [device_info_os_version] => 5.0
           [device_info_manufacturer] => Genymotion
           [device_info_brand] => generic
           [device_info_model] => Google Nexus 6 - 5.0.0 - API 21 - 1440x2560
           [device_info_app_name] => WordPress
           [device_info_app_version] => 4.1-rc-3
           [device_info_app_version_code] => 184
           [device_info_has_nfc] => false
           [device_info_has_telephony] => true
           [device_info_display_density_dpi] => 560
           [device_info_bluetooth_version] => none
           [device_info_current_network_operator] => 
           [device_info_phone_radio_type] => gsm
           [device_info_wifi_connected] => true
           [device_info_language] => en_US
```

Comment (maxme) [inline AutomatticTracks/src/main/java/com/automattic/android/tracks/DeviceInformation.java]: :+1:

**Pull request merged**